CMakeLists.txt

```diff
@@ -77,20 +77,19 @@ endif (USE_RUNPATH)
 
 if (MSVC)
   add_definitions( "/W3 /D_CRT_SECURE_NO_WARNINGS /wd4996 /wd4244 /wd4267")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+  set(CMAKE_CXX_FLAGS "/MP ${CMAKE_CXX_FLAGS}")
 else()
-  if (NOT DEFINED CMAKE_CXX_FLAGS)
```

Comment:
They get to override them now (by passing the respective -Wno or -std= flags).

I can try running cmake twice, give me a second.